src/Veldrid.Utilities/ObjParser.cs

```diff
@@ -717,22 +750,22 @@ public struct Face
             /// <summary>
             /// The first vertex.
             /// </summary>
-            public readonly FaceVertex Vertex0;
+            public FaceVertex Vertex0;
```

Comment:
Hopefully changing from readonly to normal field doesn't break compatibility. Otherwise the finalization code would need to be slightly more verbose.

Reply:
It does break compatibility, even if it is in a subtle way. How does the finalization code change?